wimpy_node:
  ros__parameters:

    opencr:
      id: 200
      baud_rate: 1000000
      protocol_version: 2.0

    wheels:
      lxy: 0.495
      radius: 0.076

    motors:
      profile_acceleration_constant: 214.577
      profile_acceleration: 0.0

